### ISSUE TITLE
FSBID Admin Persona 

### ISSUE THREAD

**Issue by krisdelacruz** [18F/State-TalentMAP#1535]
As a TalentMAP Designer
I will create a FSBID Admin persona
so that the TalentMAP team will empathize with the actors needs and pain points

## Notes
• Utilize ethnographic research to create a persona
• Provide insights about Bid Cycle management 
• Future state thinking and artifacts

**Comment by esgoodman** [comment]
@cydneycappello I'm wondering if we really need this. The "FSBid Admin" is just one person, so I'm not sure it's worth putting time into abstracting what we've learned into a persona. I'd prefer to build understanding about her job by just rolling up the main themes and lessons into text form.

**Comment by cydneycappello** [comment]
@esgoodman I totally agree with you and want to wrap up my research on this soon. It just keeps getting sidelined because of other priorities!
